Added a small howto on using Visual Studio Code debugger with Vagrant

AdamSaleh opened the pull request:
I figured it would be nice to have it in the docs :-)

cverna commented:
@AdamSaleh looks cool, you just need to sign you commit (git commit -s). For more detail you can look at the contributors guidelines (https://bodhi.fedoraproject.org/docs/developer/index.html#contribution-guidelines) and in particular the Developer Certificate of Origin part.

AdamSaleh committed:
Added a small howto for configuring the VS Code debugger with the Vagrant environment.
Works for both debugging the application running locally as well as debugging unit-tests.

Signed-off-by: Adam Saleh <asaleh@redhat.com>

ryanlerch committed:
Merge branch 'develop' into asaleh_small_doc_update

AdamSaleh committed:
Merge branch 'develop' into asaleh_small_doc_update

mergify[bot] committed:
Merge branch 'develop' into asaleh_small_doc_update

mergify[bot] committed:
Merge branch 'develop' into asaleh_small_doc_update

mergify[bot] committed:
Merge branch 'develop' into asaleh_small_doc_update
